python/smqtk/representation/key_value/postgres.py

```diff
@@ -185,10 +189,12 @@ def _py_to_bin(k):
         """
         Convert a python hashable value into psycopg2.Binary via pickle.
 
-        :param k: Python hashable type.
-        :type k: collections.Hashable
+        :param k: Python object type to be converted into a ``psycopg2.Binary``
```

Comment:
On second thought... calling it "Python object type" makes it sound like it's returning the type of an object, rather than the object itself. Let's instead say "Python object of an arbitrary type".

Reply:
Updated.